test/steps/step_login.py

```diff
@@ -0,0 +1,63 @@
+from PyQt5.QtWidgets import QDialog
+
+from PyQt5.QtTest import QTest
+from PyQt5.QtCore import Qt
+
+from behave import use_step_matcher, when, then
+
+from test.steps.step_helpers import assert_equal, assert_is_not
+use_step_matcher("re")
+
+
+@when(r'I open the login window')
+def open_login_window(context):
+    context.login_window = context.__app.login
+
+
+@when(r'I login in to the application')
+def login(context):
+    assert_is_not(context.login_window, None)
+
+    QTest.keyClicks(context.login_window.login.username, "samgomena@gmail.com")
```

Comment:
A suggestion, could we remove hardcoded value(samgomena@gmail.com) and have it read the value using a variable?